appveyor.yml

```diff
@@ -103,7 +96,7 @@ on_failure:
         Push-AppveyorArtifact "$_"
     }
 - ps: |
-    Get-ChildItem -recurse artifacts\TestsResults\Release\*.trx | `
+    Get-ChildItem -recurse artifacts\Release\TestsResults\*.trx | `
```

Comment:
Release hardcoded

Reply:
No, this is the correct path:
![image](https://user-images.githubusercontent.com/4403806/120106994-87138c00-c1a2-11eb-8fb0-5e0c1644c62a.png)